open_in_mpv/open_in_mpv.py

```diff
@@ -1,10 +1,9 @@
 # SPDX-License-Identifier: MIT
 from functools import lru_cache
-from os.path import dirname, exists, expanduser, isdir, join as path_join
-from typing import Any, Callable, Mapping, TextIO
+from os.path import dirname, exists, expanduser, expandvars, isdir, join as path_join
+from typing import Any, Final, BinaryIO, Dict, Callable, Mapping, TextIO
```

Comment:
Don't import `Dict`. Use the lowercase form: `dict[str, str]`. I will not be supporting Python before this  supported was added.